github-actions/create-pr-for-changes/lib/main.ts

```diff
@@ -284,12 +284,32 @@ async function cleanUpObsoleteBranches(
       (obsoleteBranches.length === 0 ? '-' : obsoleteBranches.join(', ')),
   );
 
+  const failedToDelete: string[] = [];
+  let deletedCount = 0;
+
   // Delete the obsolete branches.
   for (const branchName of obsoleteBranches) {
```

Comment:
Nit you don't need the `deletedCount`, as you can use `obsoleteBranches.length - failedToDelete.length`

Reply:
good point :D we can clean up in the future.